artifactory/commands/transferconfig/transferconfigmerge.go

```diff
@@ -0,0 +1,373 @@
+
+				projectConflicts, isConflict = tcc.validateProjectConflict(sourceProject, targetProject, projectConflicts)
+				conflict = conflict || isConflict
+			}
+			if !conflict {
+				err := targetServiceAccessManager.CreateProject(accessServices.ProjectParams{ProjectDetails: sourceProject})
+				if err != nil {
+					return err
+				}
+			}
+		}
+		csvPath, err := tcc.createConflictsCSVSummary(projectConflicts, time.Now())
+		if err != nil {
+			log.Error("Couldn't create the long properties CSV file", err)
+			return err
+		}
+
+		log.Info(fmt.Sprintf("Founded %d projectConflicts projects between the source service and the target, check in csv file we created for you in %s", len(projectConflicts), csvPath))
+	}
+	log.Info(coreutils.PrintTitle(coreutils.PrintBold("========== Phase 3/3 - Transferring Repositories ==========")))
+	err = tcc.mergeRepositories(sourceServiceManager, targetServiceManager)
+
+	return err
+}
+
+func (tcc *TransferConfigCommand) validateProjectConflict(sourceProject, targetProject accessServices.Project, conflicts []ProjectConflict) ([]ProjectConflict, bool) {
+	result := false
+	if sourceProject.ProjectKey == targetProject.ProjectKey || sourceProject.DisplayName == targetProject.DisplayName {
+		s := ""
+		result = true
+		if sourceProject.DisplayName != targetProject.DisplayName {
+			s = tcc.addToDifferentProperty(s, "Display Name")
+		}
+		if sourceProject.ProjectKey != targetProject.ProjectKey {
+			s = tcc.addToDifferentProperty(s, "Project Key")
+		}
+		if result {
+			if sourceProject.Description != targetProject.Description {
+				s = tcc.addToDifferentProperty(s, "Description")
+			}
+			if sourceProject.StorageQuotaBytes != targetProject.StorageQuotaBytes {
+				s = tcc.addToDifferentProperty(s, "Storage Quota Bytes")
+			}
+			if sourceProject.SoftLimit != nil && targetProject.SoftLimit != nil {
+				if *sourceProject.SoftLimit != *targetProject.SoftLimit {
+					s = tcc.addToDifferentProperty(s, "Soft Limit")
+				}
+			}
+			if sourceProject.SoftLimit == nil || targetProject.SoftLimit == nil {
+				if sourceProject.SoftLimit != nil || targetProject.SoftLimit != nil {
+					s = tcc.addToDifferentProperty(s, "Soft Limit")
+				}
+			}
+			if !tcc.checkIfSameAdminPrivilige(sourceProject.AdminPrivileges, targetProject.AdminPrivileges) {
+				s = tcc.addToDifferentProperty(s, "Admin Privileges")
+			}
+		}
+		if s != "" {
+			conflict := tcc.newConflict(sourceProject.DisplayName, targetProject.DisplayName, sourceProject.ProjectKey, targetProject.ProjectKey, s)
+			conflicts = append(conflicts, conflict)
+			return conflicts, true
+		}
+
+	}
+	return conflicts, result
+}
+
+func (tcc *TransferConfigCommand) checkIfSameAdminPrivilige(source, target *accessServices.AdminPrivileges) bool {
+	if source == nil && target == nil {
+		return true
+	}
+
+	if source == nil || target == nil {
+		return false
+	}
+
+	// if source and target Admin priviliges are not nil then we have to check all pointer admin privilige have
```

Comment:
// if source and target... --> // If the source and target...
...to check all pointer admin privilige have --> ...to check all the pointers the admin privileges includes.